website/docs/transforms.md

```diff
@@ -64,3 +64,9 @@ Set the config as:
 
 ### Custom Transformer Implementation
 You can write your own custom transformer by extending [this class](https://github.com/apache/hudi/tree/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/transform)
+
+## Related Resources
+<h3>Videos</h3>
+
+* [Learn about Apache Hudi Transformers with Hands on Lab](https://www.youtube.com/watch?v=AprlZ8hGdJo)
+* [Apache Hudi with DBT Hands on Lab.Transform Raw Hudi tables with DBT and Glue Interactive Session](https://youtu.be/DH3LEaPG6ss)
```

Comment:
Not sure if Hudi transformers are used here actually. Seem like this is generally talking about how to transform data using Hudi DBT. Remove?

Reply:
I think having linked resources to non-Hudi data transformations will be more beneficial than not having them. People will come here for any transformers even if the Hudi ones are not best fit so this could point them in the right direction.